proposals/incubation/velocitycore.md

```diff
@@ -0,0 +1,233 @@
+# VelocityCore Project Proposal
+
+## Identifier
+VelocityCore 1.0
+
+## Sponsor
+Andres Olave andres.olave@velocitycareerlabs.com
+
+## Abstract
+
+The VelocityCore is enterprise software stack for creating a "pay-to-verify"
```

Comment:
```suggestion
The [VelocityCore](https://github.com/velocitycareerlabs/velocitycore) is enterprise software stack for creating a "pay-to-verify"
```

Reply:
@hendrikebbers the code has been extracted from a far older private repo